Makefile

```diff
@@ -86,7 +86,7 @@ version:
 %: cmd/% $(TARGET_OBJS)
```

Comment:
I think the `version` should be added to `%: cmd/% $(TARGET_OBJS)` instead.

Reply:
like change below?
```
%: version  cmd/% $(TARGET_OBJS)
```

Reply:
Yes. Try if this works.

Reply:
ok I will try this change.

Reply:
ok it work in make build. For more check I will run the crowsnest with system. it maybe take 30min.
If it work I will update this.

Reply:
For now it didn't show missing `version.h` but show other.
I am still progressing. (try to install on debian 12 with crowsnest)

Current issue.
``` shell
[100%] Linking CXX static library libdatachannel-static.a
make[5]: Leaving directory '/home/pi/crowsnest/bin/camera-streamer/third_party/libdatachannel/build'
[100%] Built target datachannel-static
make[4]: Leaving directory '/home/pi/crowsnest/bin/camera-streamer/third_party/libdatachannel/build'
make[3]: Leaving directory '/home/pi/crowsnest/bin/camera-streamer/third_party/libdatachannel/build'
make[2]: Leaving directory '/home/pi/crowsnest/bin/camera-streamer/third_party/libdatachannel/build'
g++ -Werror -Wall -g -I/home/pi/crowsnest/bin/camera-streamer -D_GNU_SOURCE -Wno-error=deprecated-declarations -DUSE_HW_H264 -DUSE_FFMPEG -DUSE_LIBCAMERA -I/usr/include/libcamera  -DUSE_RTSP -I/usr/include/liveMedia -I/usr/include/groupsock -I/usr/include/BasicUsageEnvironment -I/usr/include/UsageEnvironment  -DUSE_LIBDATACHANNEL -Ithird_party/libdatachannel/include -Ithird_party/libdatachannel/deps/json/include -o camera-streamer version device/buffer.o device/buffer_list.o device/buffer_lock.o device/buffer_queue.o device/device.o device/device_list.o device/links.o output/http_ffmpeg.o output/http_h264.o output/http_hls.o output/http_jpeg.o output/output.o device/camera/camera.o device/camera/camera_debug.o device/camera/camera_decoder.o device/camera/camera_input.o device/camera/camera_isp.o device/camera/camera_output.o device/camera/camera_pipeline.o device/camera/camera_rescaller.o device/dummy/buffer.o device/dummy/buffer_list.o device/dummy/device.o device/dummy/dummy.o device/libcamera/fake_camera.o device/v4l2/buffer.o device/v4l2/buffer_list.o device/v4l2/debug.o device/v4l2/device.o device/v4l2/device_list.o device/v4l2/device_media.o device/v4l2/device_options.o device/v4l2/v4l2.o util/ffmpeg/remuxer.o util/http/http.o util/http/http_methods.o util/opts/control.o util/opts/fourcc.o util/opts/log.o util/opts/opts.o device/libcamera/buffer.o device/libcamera/buffer_list.o device/libcamera/device.o device/libcamera/libcamera.o device/libcamera/options.o output/rtsp/rtsp.o output/webrtc/webrtc.o html/control.html.o html/index.html.o html/webrtc.html.o third_party/libdatachannel/build/libdatachannel-static.a  -lpthread -lstdc++ -lavcodec -lavformat -lavutil -lcamera -lcamera-base  -lliveMedia -lgroupsock -lBasicUsageEnvironment -lUsageEnvironment  -Lthird_party/libdatachannel/build -ldatachannel-static -Lthird_party/libdatachannel/build/deps/usrsctp/usrsctplib -lusrsctp -Lthird_party/libdatachannel/build/deps/libsrtp -lsrtp2 -Lthird_party/libdatachannel/build/deps/libjuice -ljuice-static -lcrypto -lssl
/usr/bin/ld: cannot find version: No such file or directory
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:87: camera-streamer] Error 1
rm device/buffer_lock.o device/v4l2/buffer.o output/http_hls.o device/camera/camera_output.o html/webrtc.html.c device/v4l2/buffer_list.o html/webrtc.html.o device/buffer_queue.o util/ffmpeg/remuxer.o device/buffer.o html/control.html.c device/camera/camera_decoder.o device/links.o output/webrtc/webrtc.o cmd/list-devices/main.o device/v4l2/debug.o device/camera/camera_input.o util/opts/opts.o device/libcamera/device.o device/v4l2/device_options.o output/http_ffmpeg.o device/camera/camera.o device/camera/camera_rescaller.o device/libcamera/fake_camera.o html/control.html.o output/rtsp/rtsp.o cmd/camera-streamer/main.o device/buffer_list.o device/camera/camera_debug.o util/http/http.o device/camera/camera_pipeline.o device/dummy/buffer_list.o util/opts/control.o output/http_jpeg.o cmd/camera-streamer/opts.o util/opts/fourcc.o output/output.o device/v4l2/device_list.o device/device_list.o device/v4l2/device_media.o device/dummy/device.o device/libcamera/buffer_list.o device/dummy/dummy.o device/libcamera/libcamera.o util/http/http_methods.o cmd/camera-streamer/status.o device/dummy/buffer.o cmd/camera-streamer/http.o util/opts/log.o html/index.html.o html/index.html.c device/camera/camera_isp.o device/v4l2/v4l2.o device/device.o device/libcamera/options.o device/v4l2/device.o device/libcamera/buffer.o output/http_h264.o
make[1]: Leaving directory '/home/pi/crowsnest/bin/camera-streamer'
make: *** [Makefile:67: all] Error 2
make: *** [Makefile:19: install] Error 2
```

-Neko.vecter

Reply:
Ok... move this `version` in `%: version cmd/% $(TARGET_OBJS)` is not working. it will show the message below

``` shell
/usr/bin/ld: cannot find version: No such file or directory
```

So... the fix should be like the first commit I make.

Test Device:
Raspberry Pi 4B (4GB ver.) with bookworm 64 (Debian 12)

-Neko.vecter